backtesting/strategy.py

```diff
@@ -64,3 +66,33 @@ def stoploss(self, dataframe: DataFrame) -> DataFrame:
         :rtype: DataFrame
         """
         return
+
+    def join_additional_data(self, dataframe, additional, timeframe, timeframe_additional, ffill=True):
+        """
+        This function is responsible for joining the additional data to the original dataframe
+        """
+        add_df = additional.copy()
+        pair = add_df['pair'].unique()[0]
+
+        timeframe_minutes = parse_timeframe(timeframe)
+        timeframe_minutes_additional = parse_timeframe(timeframe_additional)
```

Comment:
Don't know if these variable names should contain 'minutes', since parse_timeframe() returns miliseconds.
```suggestion
        timeframe_ms = parse_timeframe(timeframe)
        timeframe_ms_additional = parse_timeframe(timeframe_additional)
```